src/Notes/PaymentsRemindMeLater.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * WooCommerce Admin Payment Reminder Me later
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+use Automattic\WooCommerce\Admin\PluginsHelper;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * PaymentsRemindMeLater
+ */
+class PaymentsRemindMeLater {
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-admin-payments-remind-me-later';
+
+	/**
+	 * Should this note exist?
+	 */
+	public static function is_applicable() {
+		return self::should_display_note();
+	}
+
+	/**
+	 * Returns true if we should display the note.
+	 *
+	 * @return Boolean
```

Comment:
Very nitpicky but to keep things consistent:
```suggestion
	 * @return bool
```